src/Cassandra/Exceptions/BusyPoolException.cs

```diff
@@ -0,0 +1,62 @@
+﻿// 
+//       Copyright DataStax Inc.
+// 
+//    Licensed under the Apache License, Version 2.0 (the "License");
+//    you may not use this file except in compliance with the License.
+//    You may obtain a copy of the License at
+// 
+//       http://www.apache.org/licenses/LICENSE-2.0
+// 
+//    Unless required by applicable law or agreed to in writing, software
+//    distributed under the License is distributed on an "AS IS" BASIS,
+//    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//    See the License for the specific language governing permissions and
+//    limitations under the License.
+// 
+
+using System.Net;
+
+namespace Cassandra
+{
+    /// <summary>
+    /// Represents a client-side error indicating that all connections to a certain host have reached
+    /// the maximum amount of in-flight requests supported.
+    /// </summary>
+    public class BusyPoolException : DriverException
+    {
+        /// <summary>
+        /// Gets the host address.
+        /// </summary>
+        public IPEndPoint Address { get; }
+
+        /// <summary>
+        /// Gets the maximum amount of requests per connection.
+        /// </summary>
+        public int MaxRequestsPerConnection { get; }
+
+        /// <summary>
+        /// Gets the size of the pool.
+        /// </summary>
+        public int ConnectionLength { get; }
+
+        private const string MessageFormat =
```

Comment:
This only seems to be used once in this class. Maybe it could be a local variable to the method or just inlined in the function call.